rtos/rtos/Thread.h

```diff
@@ -275,11 +276,13 @@ class Thread {
                      osPriority priority=osPriorityNormal,
                      uint32_t stack_size=DEFAULT_STACK_SIZE,
                      unsigned char *stack_pointer=NULL);
+    static void _thunk(const void * thread_ptr);
```

Comment:
This could just be a static function hidden in the cpp file. Doesn't need to be a member function or present in the header file.